cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -24,6 +25,10 @@ gocli=${BASE_PATH}/../cluster-up/cli.sh
 docker_prefix=localhost:$(_port registry)/kubevirt
 manifest_docker_prefix=registry:5000/kubevirt
 EOF
+
+_kubectl patch storageclass local -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+_kubectl patch storageclass csi-rbd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
```

Comment:
CI was not at all happy making the ceph provider the default storage class, if you take a look at the k8s CI run you will see a ton of failures, and all of them look like a failure when trying clean up some stuff in the pvc.